sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1273,7 +1269,26 @@ def test_sysode_linear_neq_order1_type2():
     assert dsolve(eq26) == sol26
     assert checksysodesol(eq26 , sol26) == (True , [0,0])
 
+    # Test Case added for issue #22715
+    # https://github.com/sympy/sympy/issues/22715
 
+    import sympy as sym
+
+    sym.var('a b c d e f t')
+    sym.var('x y', cls=sym.Function)
+
+    eqn1 = sym.Eq(sym.diff(x(t),t),a*x(t)+b*y(t)+c)
+    eqn2 = sym.Eq(sym.diff(y(t),t),d*x(t)+e*y(t)+f)
+    eq27 = [eqn1, eqn2]
+    sublist = list(zip((a, b, c, d, e, f), (0, -1, 10, 5, -2, 3)))
+    for idx, eqn in enumerate(eq27):
+        eq27[idx] = eq27[idx].subs(sublist)
```

Comment:
We don't need the symbols and substitutions. The code here should just directly define the equations. (Look at all of the other examples in this function)